apps/drec-ui/src/api/invitations.ts

```diff
@@ -0,0 +1,85 @@
+import {
+    useInvitationControllerGetInvitations,
+    useOrganizationControllerGetInvitationsForOrganization,
+    useInvitationControllerUpdateInvitation,
+    InvitationDTO,
+    OrganizationInvitationStatus,
+    getInvitationControllerGetInvitationsQueryKey,
+    getUserControllerMeQueryKey,
+    useUserControllerMe
+} from '@energyweb/origin-drec-api-client';
+import { useQueryClient } from 'react-query';
+import { NotificationTypeEnum, showNotification } from '@energyweb/origin-ui-core';
+
+export const useSentOrgInvitationsData = () => {
+    const { data: user, isLoading: userLoading } = useUserControllerMe();
+
+    const { data: invitations, isLoading: invitationsLoading } =
+        useOrganizationControllerGetInvitationsForOrganization(user?.organization?.id);
```

Comment:
Disable invitation fetching unless the user is there.

Reply:
Added the other role modals implementation